portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +28,37 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  /** Initially empty ArrayList. Later user comments will be stored as elements */
+  private static ArrayList<String> stringList = new ArrayList<String>();
+  static final Gson GSON = new Gson();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Difan Yang!");
+    response.setContentType("application/json;");
+    response.getWriter().println(GSON.toJson(stringList));
```

Comment:
curious, why do you still need the in memory stringList?
Basically, this could be achieved by datastore Query.